src/encoded/static/components/visualizations.js

```diff
@@ -833,8 +865,18 @@ export const ChartTable = (props) => {
                         fullResultOrgan = (fullResult && fullResult.biosample_ontology) ? fullResult.biosample_ontology.organ_slims.join(', ') : '';
                     }
 
+                    let chromatinHierarchy = [];
+                    if (props.assembly && props.assembly === 'hg19') {
```

Comment:
Can use optional chaining here just to save on typing.

Reply:
I always forget about this syntax